src/state/process.rs

```diff
@@ -44,7 +44,7 @@ enum MsgType {
 pub struct State<T: Codec, S: Codec, F: Consensus<T, S>, C: Crypto> {
     epoch_id:             u64,
     round:                u64,
-    state_machine:        SMR,
+    state_machine:        SMRHandler,
```

Comment:
rename param's name

Reply:
From state's perspective, the `SMRHandler` is the state machine.